roles/furyagent/defaults/main.yml

```diff
@@ -21,3 +21,4 @@ aws_secret_key: ""
 aws_region: "eu-west-1"
 s3_bucket_name: ""
 s3_endpooint: "http://s3-{{ aws_region }}.amazonaws.com"
+secrets_path: ../secrets
```

Comment:
Se rinominassimo questa variabile in `secrets_local_path`?

Per essere più espliciti riguardo al fatto che è un path locale alla macchine dello sviluppatore.